### PR TITLE
Azure Assistants API - Multi-Agent Example

### PR DESCRIPTION
**Pull request by Simon514771**
# Description

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure-Samples/azureai-samples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure-Samples/azureai-samples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: scenarios/Assistants/multi-agent_example/.gitignore
Comment (kdestin) [inline scenarios/Assistants/multi-agent_example/.gitignore:1]: This seems to be a copy of https://github.com/github/gitignore/blob/main/Python.gitignore

This gitignore is already included in the root gitignore of the repository

File: scenarios/Assistants/multi-agent_example/README.md
Comment (kdestin) [inline scenarios/Assistants/multi-agent_example/README.md:1]: This README.md should follow the README template as describe in the contributing guidelines

File: scenarios/Assistants/multi-agent_example/README.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# Azure OpenAI Multi-Agent Assistant API
```
Comment (kdestin) [inline scenarios/Assistants/multi-agent_example/README.md:1]: Missing YAML frontmatter for discovery on https://learn.microsoft.com

https://github.com/Azure-Samples/azureai-samples/blob/main/CONTRIBUTING.md#discoverability

File: scenarios/Assistants/multi-agent_example/README.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+)
+</pre>
+<pre lang=lisp>
+  user_proxy = client.beta.assistants.create(
+    name=name_pa,
+    instructions=instructions_pa,
+    model=api_deployment_name,
+    tools=tools
+)
+</pre>
```
Comment (kdestin) [inline scenarios/Assistants/multi-agent_example/README.md:42]: These codeblocks seem wrong?

`<pre></pre>` doesn't seem to render syntax highlighting on github, and the code in the block isn't lisp.

File: scenarios/Assistants/multi-agent_example/README.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+                            tool_responses.append({"tool_call_id": call.id, "output": tool_response})
+</pre>
+
+<pre lang=lisp>
+  run = client.beta.threads.runs.submit_tool_outputs(
+        thread_id=thread.id,
+        run_id=run.id,
+        tool_outputs=tool_responses
+      )
+</pre>
```
Comment (kdestin) [inline scenarios/Assistants/multi-agent_example/README.md:134]: These codeblocks seem wrong?

`<pre></pre>` doesn't seem to render syntax highlighting on github, and the code in the block isn't lisp.

File: scenarios/Assistants/multi-agent_example/AssistantAPI_MultiAgent_Example_v1.1.ipynb
Comment (kdestin) [inline scenarios/Assistants/multi-agent_example/AssistantAPI_MultiAgent_Example_v1.1.ipynb:1]: This notebook doesn't adhere to the template, as described in the contributing guidelines

https://github.com/Azure-Samples/azureai-samples/blob/main/CONTRIBUTING.md#write-your-contribution

**Review by kdestin — CHANGES_REQUESTED** [review]
Thank you for your contribution!

**Comment by kdestin** [comment]
Consider renaming:

* `scenarios/Assistants/multi-agent_example` to `scenarios/Assistants/multi-agent`
* `AssistantAPI_MultiAgent_Example_v1.1.ipynb` to `multi-agent.ipynb`

To reduce redundancy and keep the paths shorter.

File: scenarios/Assistants/multi-agent_example/README.md (hunk 3 above)
Comment (Simon514771) [reply scenarios/Assistants/multi-agent_example/README.md]: Address in lastest commite

**Comment by Simon514771** [comment]
All changes have been addressed. Pre-commit was changing certain code formatting that was important for the teaching purposes of the sample, so wer e overridden.

**Comment by kdestin** [comment]
> All changes have been addressed. Pre-commit was changing certain code formatting that was important for the teaching purposes of the sample, so wer e overridden.

@Simon514771 If it's imperative that some of the python code in the sample is not reformatted by the code formatter, you can refer to [this documentation](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#ignoring-sections) for how to disable formatting for the relevant section of code.

**Pull request closed without merging**